docs/toc.yml

```
@@ -1289,30 +1289,21 @@ items:
     - name: Use search options to find text in your Word add-in
       href: word/search-option-guidance.md
       displayName: Word
-
-- name: Patterns and practices
+- name: Samples
   items:
-  - name: Overview
-    href: overview/patterns-and-practices-overview.md
+  - name: Samples list
+    href: overview/office-add-in-code-samples.md
```

you can also add displayName and list pnp so it can show when ppl search in the title search bar
```suggestion
    href: overview/office-add-in-code-samples.md
    displayName: PnP
```